### PR TITLE
Incorporate ProtocolState-Fuzzer

### PR DESCRIPTION
**Pull request by pfg666**
I believe we can merge this to master. The only inconvenience is that our `install.sh` script has to do more work. 

1. we have to download ProtocolState-Fuzzer/set it up (this will be fixed once ProtocolState-Fuzzer is published to Maven)
2. we have to patch ProtocolState-Fuzzer for Java 11 compatibility (this will hopefully be unnecessary once we upgrade to a version 5.x.y of TLS-Attacker)

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -40,10 +50,40 @@ function check_mvn() {
...
+            mvn install
+        )
+    fi
+}
+
 # Check for the presence of Java and maven and if 'apt-get' is present, install them
 check_java
 check_mvn
 
+# Install ProtocolState-Fuzzer before patching for Java 11
```
Comment (kostis) [inline install.sh:85]: The comment here is misleading, I think. It should be deleted or changed.
Both patching for Java 11 and installation happen by the same function and patching takes place first.

Comment (pfg666) [reply install.sh]: Fixed!

File: src/test/java/se/uu/it/dtlsfuzzer/MainTest.java
Comment (kostis) [inline src/test/java/se/uu/it/dtlsfuzzer/MainTest.java:1]: Perhaps we can just remove this code, rather than leaving it commented out.

**Review by kostis — APPROVED** [review]
Looks OK overall.  We can merge.

**Comment by pfg666** [comment]
I made some additional changes. Check them and then we merge!

**Review by kostis — APPROVED** [review]

**Pull request merged**